_includes/1.0/api/creditmemo.html

```diff
@@ -66,7 +66,7 @@ <h3 id="creditmemojsonformat">JSON Format</h3>
                 <td>{{ site.format.type.integer }}</td>
                 <td>no</td>
                 <td>no</td>
-                <td>Id of the <a href="/docs/1.0/api/#contact" title="">Contact</a> to bill.<br />Should belong to that invoice's Customer.</td>
+                <td>Id of the <a href="/docs/1.0/api/#contact" title="">Contact</a> to credit memo.<br />Should belong to that invoice's Customer.</td>
```

Comment:
Wrong edit. 'to bill' means 'à facturer', you didn't need changing that one.

What needed to be changed was the end of the comment: `invoice's customer` into `credit memo's customer`